rllib/algorithms/crr/crr.py

```diff
@@ -38,19 +36,9 @@ def __init__(self, algo_class=None):
         self.advantage_type = "mean"
         self.n_action_sample = 4
         self.twin_q = True
-        self.target_update_grad_intervals = 100
+        self.target_network_update_freq = 100
```

Comment:
Can you update the default to 100*batch_size? 100 would break things.

Reply:
done

Reply:
done